WordPress/Classes/Extensions/String+Helpers.swift

```diff
@@ -30,4 +30,13 @@ extension String {
         }
         return self
     }
+
+    /// Returns a string without the character at the specified index.
+    /// This is a non-mutating version of `String.removeAtIndex(_:)`.
+    func removingCharacterAtIndex(index: Index) -> String {
```

Comment:
I'm living in the future 😉 https://swift.org/documentation/api-design-guidelines/#name-according-to-side-effects
That said, in Swift 3 the mutating method will be `String.remove(at:)`, so this should be `String.removing(at:)`